ci/windows/build.cmd

```diff
@@ -1,6 +1,14 @@
-set PATH=%PATH%;C:\Program Files\CMake\bin
+call "c:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
```

Comment:
Just a minor suggestion: comment these ultra-windowsy bits (also below) a bit more?  You're saying this puts stuff in the path, but maybe there's a handy docs link for it or some such, for when we need to revisit. Etc.

Any chance there's a way to make the path to it less hardwired? (Perhaps that's normal, I really don't know this stuff!)